### ISSUE TITLE
No response on 8080

### ISSUE THREAD

**Issue by rajnisheu** [greenmail-mail-test/greenmail#379]
Hi,

Using the provided example, there's no response on 8080. Is that expected? 
Am just using the defaults. Providing here the docker-compose.yml which is from one of the clients/example
```
version: '2'
services:
  greenmail:
    image: greenmail/standalone:1.6.0-SNAPSHOT
    environment:
    # Enable GreenMail verbose mode
    - GREENMAIL_OPTS=-Dgreenmail.setup.test.all -Dgreenmail.hostname=0.0.0.0 -Dgreenmail.auth.disabled -Dgreenmail.verbose
    ports:
    - "3025:3025"
    - "3110:3110"
    - "3143:3143"
    - "3465:3465"
    - "3993:3993"
    - "3995:3995"
    - "8080:8080"

**Comment by marcelmay** [comment]
@rajnisheu , the API is only available with version[ 1.6.2+](https://greenmail-mail-test.github.io/greenmail/#deployment). You're using old 1.6.0 and SNAPSHOT, so I recommend to update to latest 1.6.3 .

I'll update the client examples versions. Thx!

**Comment by rajnisheu** [comment]
Wonderful @marcelmay! 
Many thanks for the quick response :)
